extensions/3rdparty/ArrayExtension/ArrayExtension.php

```diff
@@ -442,7 +442,7 @@ function arraysort( &$parser, $arrayid , $sort = 'none') {
 *  merge values two arrayes identified by arrayid1 and arrayid2 into a new array identified by arrayid_new.
 *  this merge differs from array_merge of php because it merges values.
 */
-    function arraymerge( &$parser, $arrayid_new, $arrayid1, $arrayid2='' ) {
+    function arraymerge( $parser, $arrayid_new, $arrayid1, $arrayid2='' ) {
         if (!isset($arrayid_new) )
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add curly braces around the nested statement(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS121)